README.md

```diff
@@ -368,7 +368,7 @@ Parameters of the driver script, `abyss-pe`
  * `q`: minimum base quality [`3`]
  * `s`: minimum unitig size required for building contigs (bp) [`200`]
  * `S`: minimum contig size required for building scaffolds (bp) [`s`]
- * `t`: minimum tip size (bp) [`2k`]
+ * `t`: maximum length of dangling edges to trim [`k`]
```

Comment:
I prefer the term "blunt contig" (or "tip") to "dangling edge". In ABySS parlance, *k*-mers or contigs are vertices and sequence overlaps are edges.